blueprints/ember-responsive/files/app/breakpoints.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  mobile:  '(max-width: 768px)',
+  tablet:  '(min-width: 769px) and (max-width: 992px)',
+  desktop: '(min-width: 993px) and (max-width: 1200px)',
+  jumbo:   '(min-width: 1201px)'
```

Comment:
I copy / pasted this out of the readme. Happy to use whatever we want. In my app, I don't even have a "jumbo" breakpoint, so I could also just remove this and leave mobile, tablet, desktop, which I feel are pretty universal.